### PR TITLE
reordered Company portal doc setup steps

### PR DESCRIPTION
**Pull request by hathin**
added more details and reordered steps to follow current process.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Comment by yogkumgit** [comment]
@hathin

File: microsoft-365/managed-desktop/get-started/deploy-apps.md [hunk 1]
```diff
@@ -55,17 +55,18 @@ If you haven't signed up with Microsoft Store for Business, you can sign up when
 3. Use Search to find that the app that you want, and select the app.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-started/deploy-apps.md]:
```suggestion
3. Use Search to find the app that you want, and select the app.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hathin, not related to the PR, but kindly also commit the suggested change to correct the typo in line 55. Many thanks.

**Comment by ghost** [comment]
[CLA assistant check](https://cla.opensource.microsoft.com/MicrosoftDocs/microsoft-365-docs?pullRequest=6571) <br/>All CLA requirements met.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**